lib/services/build.ts

```diff
@@ -176,7 +176,17 @@ export class BuildService implements Project.IBuildService {
 					}
 				} else if(!settings.buildForiOSSimulator) {
 					let deviceIdentifier = settings.device ? settings.device.getIdentifier() : undefined;
-					provisionData = this.$identityManager.autoselectProvision(appIdentifier, settings.provisionTypes, deviceIdentifier).wait();
+					try {
+						provisionData = this.$identityManager.autoselectProvision(appIdentifier, [constants.ProvisionType.AdHoc], deviceIdentifier).wait();
+					} catch (error) {
+						this.$logger.warn("Cannot find suitable AdHoc provision so no QR code will be generated");
```

Comment:
"Cannot generate QR code because an applicable AdHoc provisioning profile is not available.", as @ikoevska suggested.